hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/diskUsage/diskUsage.tsx

```diff
@@ -539,7 +539,10 @@ export class DiskUsage extends React.Component<Record<string, object>, IDUState>
     return (
       <div className='du-container'>
         <div className='page-header'>
-          Disk Usage
+          Disk Usage&nbsp;&nbsp;
+          <Tooltip placement="rightTop" title="Shows Disk Usage Info for FSO buckets Only">
```

Comment:
LGTM +1, however pls add a task in [HDDS-7810](https://issues.apache.org/jira/browse/HDDS-7810)  to remove back this tool tip once [HDDS-7810](https://issues.apache.org/jira/browse/HDDS-7810) is resolved along with it.

Reply:
@dombizita 
Created  SubTask under this JIRA https://issues.apache.org/jira/browse/HDDS-5305